.github/actions/ginkgo/main-k8s-versions.yaml

```diff
@@ -1,12 +1,19 @@
 # This file contains which kernel versions should be run with which k8s versions
 ---
 include:
+  - k8s-version: "1.29"
+    ip-family: "dual"
+    # renovate: datasource=docker
+    kube-image: "quay.io/cilium/kindest-node:v1.29.0-rc.1@sha256:6631d58da3569930cf21697c2113feb9c76bc044c1c13d98808a3695dd91d8b0"
+    # renovate: datasource=docker depName=quay.io/lvh-images/kind
+    kernel: "bpf-next-20231127.012907@sha256:e46bb7809049c1408637ba477c31fb69781014a30af66fa36ba67df72f4d1ed6"
+
   - k8s-version: "1.28"
     ip-family: "dual"
     # renovate: datasource=docker
     kube-image: "kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31"
     # renovate: datasource=docker depName=quay.io/lvh-images/kind
-    kernel: "bpf-next-20231127.012907@sha256:e46bb7809049c1408637ba477c31fb69781014a30af66fa36ba67df72f4d1ed6"
+    kernel: "4.19-20231124.100406@sha256:66bb0396f461b845c62de678cc9b9598f08c0fb0080139a9830dc9e7992952d6"
```

Comment:
4.19 will be EOL soon, so better we can run with something else now.

Reply:
There were internal discussions about that, and I don't think we'll drop support “soon”. Makes sense to me to keep running 4.19 for now.

Reply:
One fun fact, from where I am, summer is just 2 days away :australia: :hot_face: :rofl:

Reply:
@sayboras I don't understand this comment :sweat_smile:

Reply:
ah I misread that 4.19 is EOL in _coming_ summer release, and summer is almost there for me :sweat:.

Reply:
I don't think it's coming “next summer” in either hemisphere, for what it's worth :sweat_smile: